.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: python
 
+notifications:
```

Comment:
Ah, forgot to remove it! Thanks for having checked. It has been removed as requested.